### DISCUSSION TITLE
Viewing installed plugins in Flow

### DISCUSSION THREAD

**Discussion by LBF38** [Flow-Launcher/Flow.Launcher#1448] [Ideas]
Hi devs !
I love your Flow Launcher and I'm using it all the time !

However, it misses seeing the plugins installed in the settings panel of the app launcher.

I imagine it as simple as a check mark on the bottom right corner of each plugin in the plugin store.

What do you think of this feature ?

**Comment by jjw24** [comment]
Hi @LBF38, good to hear 👍 

For displaying the installed plugins we should already have something in the works that displays a label in the plugin store for the installed plugins.

**Reply by LBF38** [reply]
Great! Thank you for the quick reply!
